src/main/java/core/basesyntax/io/write/impl/ReportWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.io.write.impl;
+
+import core.basesyntax.io.write.ReportWriter;
+
+public class ReportWriterImpl implements ReportWriter {
+    private final ReportWriter writer;
+
+    public ReportWriterImpl(ReportWriter writer) {
+        this.writer = writer;
+    }
+
+    @Override
+    public void write(String report, String reportPath) {
+        writer.write(report, reportPath);
+    }
+}
```

Comment:
The `ReportWriterImpl` class is not independent as it relies on another `ReportWriter` instance. According to the checklist, services should be independent. You should implement the writing logic directly in this class or use dependency injection to provide the `ReportWriter` if this wrapper approach is part of a design pattern that's not yet clear from the current context.